tk_barrydegraaff_zimbra_openpgp/tk_barrydegraaff_zimbra_openpgp.js

```diff
@@ -109,7 +116,24 @@ tk_barrydegraaff_zimbra_openpgp.prototype.init = function() {
          this.setUserProperty("zimbra_openpgp_options", JSON.stringify(tk_barrydegraaff_zimbra_openpgp.settings), true);
          tk_barrydegraaff_zimbra_openpgp.prototype.localStorageSave(tk_barrydegraaff_zimbra_openpgp.settings['aes_password'], localStorage['zimbra_openpgp_privatekey'+tk_barrydegraaff_zimbra_openpgp.prototype.getUsername()]);
       }
-   }      
+   }
+   
+   var zimbra_openpgp_privatepass = this.getUserProperty("zimbra_openpgp_privatepass");
+   if ((zimbra_openpgp_privatepass.length > 0)  && (zimbra_openpgp_privatepass.indexOf('-cryptedpp-') < 1))
+   {
+      //found a zimbra_openpgp_privatepass on server that was stored in a previous version, encrypt it
+      var encryptedPassphrase = Aes.Ctr.encrypt('-----openpgppassphrase-----='+zimbra_openpgp_privatepass, tk_barrydegraaff_zimbra_openpgp.settings['aes_password'], 256);
+      if (tk_barrydegraaff_zimbra_openpgp.settings['store_passphrase_locally'] == 'true')
+      {
+         localStorage['zimbra_openpgp_privatepass'+tk_barrydegraaff_zimbra_openpgp.prototype.getUsername()] = '---cryptedpp---' + encryptedPassphrase;
+         this.setUserProperty("zimbra_openpgp_privatepass", '', false);
```

Comment:
the last argument should be `true` not `false`, when saving multiple UserProperties, only the last one should be set to `true`. Doing this will combine as much UserProperties in a single network request.

However, when only storing one UserProperty, it must always be true. 

Because when set to `false` the zimlet will not send any UserProperty update request to the server, untill one `true` comes... in this case it may never come.... = bug